src/Utils/View.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Masterclass\Utils;
+
+use Masterclass\Exceptions\BadViewDataException;
+
+class View
+{
+    /**
+     * @param string $__view__ The name of the view
+     * @param array $__data__ an array of data
+     * @return mixed
+     * @throws BadViewDataException
+     */
+    public static function make($__view__, $__data__ = [])
```

Comment:
I used these variable names so as to not collide with the variables being passed in in the data array because it is using `extract`, I am running under the assumption that we are going to be swapping this out with a proper template manager in the future